src/runtime/runtime_wasm_wasi.go

```diff
@@ -1,4 +1,4 @@
-//go:build tinygo.wasm && (wasi || wasip1)
```

Comment:
Should this be renamed to `runtime_wasm_wasip1.go`?

Reply:
@ydnar Yes, eventually.  That will be part of the "rip out the duplicated code" bit I'm still working on.

Reply:
@dgryski is this still something needed to be done?

Reply:
Pretty sure this file doesn't exist anymore.